github/settings/settings.ts

```diff
@@ -1,16 +1,39 @@
 import { ISetting, SettingType } from '@rocket.chat/apps-engine/definition/settings';
 
-export enum AppSettings{
-    ReminderCRONjobString = 'reminderCronJobString'
+export enum AppSettings {
+    ReminderCRONjobID = 'reminder_cron_job_id',
+    ReminderCRONjobLabel = 'cron-job-string-for-pr-reminders',
+    ReminderCRONjobPackageValue = '0 9 * * *',
+    BaseHostID = "base_host",
+    BaseHostLabel = "base-host",
+    BaseHostPackageValue = "https://github.com/",
+    BaseApiHostID = "base_api_host",
+    BaseApiHostLabel = "base-api-host",
```

Comment:
don't make things like `-` and `_` be the differentiator between an id and a label. Just add a label `_label` or `-label` .. it's confusing.

Reply:
Ohk I will change it thanks for valuable suggestion @debdutdeb